README.md

```diff
@@ -1,2 +1,220 @@
+---
+
+The Asgardio Tomcat OIDC Agent enables you to add OIDC-based login, logout to your Apache Tomcat web apps with
+ minimum hassle.
+
+
+- [Getting Started](#getting-started)
+- [How it works](#how-it-works)
+- [Integrating Asgardio Tomcat OIDC Agent](#integrating-asgardio-tomcat-oidc-agent)
+  * [To your existing webapp](#to-your-existing-webapp)
+  * [To your Java source project](#to-your-java-source-project)
+- [Building from the source](#building-from-the-source)
+- [Contributing](#contributing)
+  * [Reporting issues](#reporting-issues)
+- [License](#license)
+
+## Getting started
+
+You can experience the capabilities of Asgardio Tomcat OIDC Agent by following this small guide which contains main
+ sections listed below.
+
+  * [Prerequisites](#prerequisites)
+  * [Configuring the sample](#configuring-the-sample)
+  * [Configuring Identity Server](#configuring-identity-server)
+  * [Running the sample](#running-the-sample)
+
+### Prerequisites
+1. WSO2 Identity Server and it's [prerequisites](https://is.docs.wso2.com/en/next/setup/installing-the-product/).
+
+A sample app for demonstrating OIDC based authentication/authorization, logout and attribute retrieval is hosted at:
+[TODO link]
+
+You can download the pre-built oidc-sample-app.war from [TODO link]
+
+### Configuring the sample
+1.  Add the following entry to the `/etc/hosts` file of your machine to configure the hostname.
+   ```
+   127.0.0.1 localhost.com
+   ```
+
+### Configuring Identity Server
+
+Here we are using WSO2 Identity Server as the OpenID Provider. The sample can be configured with any other preferred
+ OpenID Provider as well.
+ 
+1. Start the WSO2 IS. 
+2. Access WSO2 IS management console and create a service provider (ex:- oidc-sample-app)
+   
+   For the service provider, configure Oauth/OpenID Connect under Inbound Authentication Configuration. In this
+    configuration,
+   use following parameters and options,
+     
+       Callback URL - http://localhost:8080/oidc-sample-app/oauth2client
+
+
+   Keep the other default settings as it is and save the configuration.
+   
+   Next, expand the [Claim Configuration](https://is.docs.wso2.com/en/latest/learn/configuring-claims-for-a-service-provider/#configuring-claims-for-a-service-provider) section. In this configuration, Set the following config and add the claims you 
+   need to retrieve (ex: http://wso2.org/claims/lastname) from the web app.
+   
+       Select Claim mapping Dialect - Use Local Claim Dialect
+       
+   See the example claim config below.
+   ![Claim Config](https://user-images.githubusercontent.com/15249242/90488235-38d45580-e159-11ea-8beb-52d6b5c35034.png)
+
+### Running the sample
+
+1. Deploy the application, `oidc-sample-app.war` using Apache Tomcat.
+2. Try out the application by accessing the `http://localhost:8080/oidc-sample-app/index.html`.
```

Comment:
localhost.com

Reply:
Resolved in https://github.com/asgardio/asgardio-tomcat-oidc-agent/pull/7/commits/bf4f7fa7367a16bd78c9cbabc954df5fb7451e18